### PR TITLE
Create styled reusable button component

### LINKED ISSUE #30: Create reusable button component and style it
We need a single button that can be used on the three pages (views).
AC:
- The button should look like the one in [figma design](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44?node-id=0%3A1)
- The button should be reusable.

Notes: 
- by reusable I mean that it should not has hard-coded values. For example, the button's text should be passed as props and not hard-coded.

### PR DESCRIPTION
**Pull request by JimeBlue**
## Description
This PR styles the buttons on the three pages, the `home page`, the ` add an item` page and the `list page`, when the list is empty.

- The `Button.js` component was created.
- I passed the `width` of the buttons as prop, since there are two versions, a small and a larger one.
- Used different Tailwind.css features to style the button.
- I included the hover state.
- Added a focus state to account for accessibility.
- Imported the `font-family` in `index.css` and extended it in `tailwind-config.js`.

### Notes:
- I change the font-family from **Roboto** to **Roboto-Flex** and the wording of the buttons. I think that makes them look better as well as more readable. 

## Related Issue
closes #30

## Acceptance Criteria

- [x] The buttons should look like the ones in [figma design](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44?node-id=0%3A1)

-  [x] The `Button` component should be reusable.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

before-1

before2

### After

after1
after2

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with ` git pull origin jm-rb-to-create-styled-reusable-button` and check that branch out with ` git checkout jm-rb-to-create-styled-reusable-button`.
- Compare the buttons to the design in Figma.

### CODE REVIEW

**Review by redapy — APPROVED** [review]
Great job Jimena, LGTM!

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Button = ({ text, disabled, createToken, width }) => {
+  return (
+    <button
+      className={`${width} bg-yellow-400 text-black text-sm font-semibold uppercase px-4 py-4 rounded-full font-button hover:bg-transparent outline outline-yellow-400 hover:outline-yellow-400 focus:outline-4`}
```
Comment (hellodeborahuk) [inline src/components/Button.js]: I like that you've created a button component to reuse. If you have `px-4` and `py-4`, you can just use `p-4`. But I wonder if it looks a little tall, maybe `py-3` or 2.5 would look better? And `text-md`?

**Review by hellodeborahuk — COMMENTED** [review]
Great work Jimena!

File: src/components/Button.js (hunk 1 above)
Comment (JimeBlue) [reply src/components/Button.js]: Thanks for your suggestion! I´ll try that out 😃

**Pull request merged**